packages/relay-experimental/useRelayEnvironment.js

```diff
@@ -11,7 +11,7 @@
 
 'use strict';
 
-const ReactRelayContext = require('react-relay/ReactRelayContext');
+const {ReactRelayContext} = require('react-relay');
```

Comment:
the problem here is that we don't actually want to bring in all of 'react-relay' in our web bundle, let me see if I can get this to work

Reply:
Hadn't considered this aspect. 🤔

Reply:
I think we still want to be compatible with the apis in `react-relay`, so you can use both apis together and migrate incrementally to hooks if you want (without having to convert everything to hooks at once), which means that we need to use the same context; but for cases where we're exclusively using relay-experimental we don't to bring in all of react-relay.

I think I might have a solution for this though

Reply:
Okay. Looking forward to being blown away. 😉